README.md

```diff
@@ -146,7 +146,7 @@ $ sudo dnf install dosfstools dpkg expect gpg parted python3-PyYAML python3-setu
 Finally, return to the project directory and run
 
 ```
-$ sudo python3 setup.py install
+$ sudo pip3 install pieman
```

Comment:
it gives me:
sudo: pip3: command not found

Reply:
The following solves the above issue:
$ sudo apt-get install python3-pip

Let's add python3-pip to the list of packages need to be installed on Debian or Ubuntu (README.md)

Reply:
Agree.